frontend/public/locales/fr/learn.json

```
@@ -3,70 +3,70 @@
   "explore": "(FR) Explore our content",
   "breadcrumbs": {
     "home": {
-      "text": "(FR) Retirement Hub",
+      "text": "Carrefour retraite",
       "link": "/"
     }
   },
   "banner": {
-    "title": "(FR) Not sure where to begin? Take the quiz!",
-    "text": "(FR) Take our quiz and receive a personalized checklist to help guide you through retirement.",
-    "quiz": "(FR) Take the quiz now!"
+    "title": "Vous ne savez pas par où commencer. Répondez à notre questionnaire!",
+    "text": "Recevez une liste de vérification personnalisée pour vous guider tout au long de votre retraite.",
+    "quiz": "Répondez au questionnaire maintenant!"
   },
   "sections": [
     {
-      "title": "(FR) Managing your money in retirement",
-      "body": "(FR) Prepare for a comfortable retirement by exploring our money management section, which includes guidance on personal savings, identifying potential income sources, tax implications, and more - all with CPP and OAS benefits in mind.",
+      "title": "Gérer votre argent à la retraite",
+      "body": "Préparez votre vieillesse en examinant les différentes sources de revenus de retraite et apprenez à gérer votre argent une fois à la retraite.",
       "cards": [
         {
-          "title": "(FR) Main sources of retirement income",
-          "read": "(FR) 5 MIN READ",
-          "body": "(FR) Learn about public & private pensions, personal savings and investments.",
+          "title": "Principales sources de revenus de retraite",
+          "read": "Temps de lecture : 9 minutes",
```

```suggestion
          "read": "Temps de lecture\u00a0: 9 minutes",
```

(to prevent wrapping at the colon)